tendrl/node_agent/flows/import_cluster/__init__.py

```diff
@@ -37,13 +39,32 @@ def run(self):
 
                     tendrl_ns.etcd_orm.client.write("/queue/%s" % uuid.uuid4(),
                                                json.dumps(job))
-
-
+        Event(
```

Comment:
I think you need to add a status for creating the jobs for other nodes in the cluster(line 31)?